src/panels/config/integrations/integration-panels/matter/matter-config-panel.ts

```diff
@@ -155,6 +170,34 @@ export class MatterConfigPanel extends LitElement {
     }
   }
 
+  private _redirectOnNewDevice() {
+    if (this._unsubDevReg) {
+      return;
+    }
+    let lastSeen: Set<string> | undefined;
+
+    this._unsubDevReg = subscribeDeviceRegistry(
```

Comment:
You can also use `hass.devices`

Reply:
Ooof I'm old. We should not allow any new use of `subscribeDeviceRegistry`.